util/reflect_test.go

```diff
@@ -145,6 +143,11 @@ func TestIsValueFuncs(t *testing.T) {
 			okValues: []interface{}{testStruct},
 		},
 		{
+			desc:     "IsValueInterface",
+			function: IsValueInterface,
+			okValues: []interface{}{},
+		},
+		{
```

Comment:
Consider adding something that covers `IsValueInterfaceToStructPtr` here, which is currently uncovered but should be relatively easy to add?

Reply:
Done.